recipes/alcor/meta.yaml

```diff
@@ -10,7 +10,9 @@ package:
 build:
    # Passes some invalid flags for clang
    skip: True  # [osx]
-   number: 2
+   number: 3
```

Comment:
Better wait for https://github.com/bioconda/bioconda-recipes/pull/46388 to be merged